components/org.wso2.carbon.identity.application.authenticator.push/src/main/java/org/wso2/carbon/identity/application/authenticator/push/PushAuthenticator.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Reply:
2019 is the correct year for the license. The class was initially created in 2019